lib/matplotlib/patheffects.py

```diff
@@ -373,3 +375,170 @@ def draw_path(self, renderer, gc, tpath, affine, rgbFace):
         if clip_path:
             self.patch.set_clip_path(*clip_path)
         self.patch.draw(renderer)
+
+
+class TickedStroke(AbstractPathEffect):
+    """
+    A line based PathEffect which draws a path with a ticked style.
+    This line style is frequently used to represent constraints in
+    optimization.  The ticks indicate that one side of the line is
+    invalid.
```

Comment:
"...or to represent a closed boundary of a domain (i.e. a wall or the edge of a pipe)."

Reply:
OK